tests/integration-tests/iOS/iOS.Helix.SDK.Tests.proj

```diff
@@ -2,6 +2,9 @@
   <Import Sdk="Microsoft.DotNet.Helix.Sdk" Project="Sdk.props"/>
 
   <PropertyGroup>
+    <!-- Workaround changes from newer MSBuild requiring additional properties set (otherwise gets a weird error about targeting desktop 4.0) --> 
+    <TargetFrameworkVersion>5.0</TargetFrameworkVersion>
+    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
```

Comment:
@premun I think this (variant of what was done in https://github.com/dotnet/arcade/pull/5996/files ) should unblock the build, unsure if you wanted to keep the stuff you had already modified though?

Reply:
Reviewed and it seems reasonable, so I clicked approve and once it built clean the bot merged :)